internal/csi-addons/rbd/volumegroup.go

```diff
@@ -614,6 +615,23 @@ func (vs *VolumeGroupServer) ModifyVolumeGroupMembership(
 		return nil, getGRPCError(err)
 	}
 
+	// Need to add scheduling again, as there might be a race condition
+	// where the python scheduler might run at the time when the mirroring
+	// was disabled on the group and it will remove the schedule
```

Comment:
```suggestion
	// Need to add scheduling again since disabling mirroring may have 
	// removed snapshot schedule too.
```
Let's just treat it as a madatory step for our hack of disabling and enabling mirroring.

https://github.com/red-hat-storage/ceph-csi/blob/aaccb5f26955cd6db04eed43411f6bf94599951d/internal/csi-addons/rbd/replication.go#L473-L485

Reply:
Done